### PR TITLE
Merge develop

### PR DESCRIPTION
**Pull request by yavrsky**

### CODE REVIEW

File: test/utils/deploy/contractManager.ts [hunk 1]
```diff
@@ -3,6 +3,10 @@ import { ContractManagerContract } from "../../../types/truffle-contracts";
 const ContractManager: ContractManagerContract = artifacts.require("./ContractManager");
 
 export async function deployContractManager() {
+    if (await web3.eth.getCode("0x1820a4B7618BdE71Dce8cdc73aAB6C95905faD24") === "0x") {
```
Comment (DimaStebaev) [inline test/utils/deploy/contractManager.ts:6]: You don't need to deploy 1820 here. It is deployed in migrations.

File: test/utils/deploy/nodesData.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ const deployNodesData: (contractManager: ContractManagerInstance) => Promise<Nod
                             },
                             async (contractManager: ContractManagerInstance) => {
```
Comment (DimaStebaev) [inline test/utils/deploy/nodesData.ts]: For initializers with default parameters set all this function can be removed.

File: test/utils/deploy/contractManager.ts (hunk 1 above)
Comment (yavrsky) [reply test/utils/deploy/contractManager.ts:6]: it's needed for tests

Comment (DimaStebaev) [reply test/utils/deploy/contractManager.ts:6]: Migrations are run before each test file so you don't need it for tests.

Comment (yavrsky) [reply test/utils/deploy/contractManager.ts:6]: with truffle - yes, buidler - no

**Review by DimaStebaev — APPROVED** [review]

**Pull request merged**